sdcm/nemesis.py

```diff
@@ -258,7 +258,7 @@ def get_node_info_list(verification_node):
                 self._terminate_cluster_node(self.target_node)
                 # Replace the node that was terminated.
                 if add_node:
-                    self._add_and_init_new_cluster_node()
+                    self._add_and_init_new_cluster_node(timeout=10800)
```

Comment:
actually instead of all these changes we could just increase the default timeout in wait_db_up

Reply:
But then all nodes could wait for 3 hours and we don't want this to happen without to know about it.